shared/fight.cpp

```diff
@@ -234,13 +235,20 @@ fightable setup_fightable(int id) {
     return fight_character; // RVO
 }
 
+template fightable setup_fightable<nibashared::client_staticdata>(int id);
```

Comment:
as below, we move it back to .h then this "explicit instantiation" could be removed